Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeInteractionRegionLayers.mm

```diff
@@ -86,7 +87,11 @@ void updateLayersForInteractionRegions(CALayer *layer, RemoteLayerTreeHost& host
     ASSERT(properties.changedProperties & RemoteLayerTreeTransaction::EventRegionChanged);
 
     HashMap<IntRect, CALayer *> interactionRegionLayers;
+    CALayer *tileGridLayer = nil;
     for (CALayer *sublayer in layer.sublayers) {
+        if (!tileGridLayer && String(sublayer.name) == TileController::tileGridContainerLayerName())
```

Comment:
@hortont424 instead of tileGridContainerLayer I noticed that RenderLayerBacking has a clippingLayer() member function. I don't know if I should look for that instead.

I know we discussed using the GraphicsLayer tree but that would involve moving this out of the UI process right?

Reply:
Not moving it out of the UIP *exactly* (it will still end up in the UIP by way of UI-side compositing), just having GraphicsLayer sublayers (PlatformCALayers) for interaction regions interleaved correctly with the content layers. Not sure it's worth it, but it's probably the "right" thing. Maybe see what @smfr thinks.

Reply:
Hunting by the tile grid layer by name seems very wrong, though.